src/MoBi.Engine/Sbml/UnitDefinitionImporter.cs

```diff
@@ -14,23 +14,28 @@ namespace MoBi.Engine.Sbml
 {
    internal class CombinedUnit
    {
-      public string Name { get => (_direct?.Name ?? "1") + (_inverse != null ? $"/{_inverse.Name}" : ""); }
+      public string Name => (_direct?.Name ?? "1") + (_inverse != null ? $"/{_inverse.Name}" : "");
+
       public double Rate { get; private set; } = 1;
+
+      private Unit _direct;
+      private Unit _inverse;
+
       public void AddUnit(int kind, double exponent, double multiplier, double scale, IDictionary<int, Unit> baseUnitsDictionary)
       {
          if (!baseUnitsDictionary.ContainsKey(kind))
             return;
+      
          if (exponent < 0)
             _inverse = baseUnitsDictionary[kind];
          else
             _direct = baseUnitsDictionary[kind];
+
          Rate *= multiplier * Math.Pow(10, scale);
       }
-      private Unit _direct { get; set; }
```

Comment:
private menber. Why a get set?

Reply:
Not really needed, sometimes I do this for finding easier where it is been used. So even when it is private, it is a full property and you can easily navigate but you are right, not necessary.

Reply:
Resharper navigates with private members...plus it follows the rest of the code